modules/prediction/proto/feature.proto

```diff
@@ -9,13 +9,13 @@ import "modules/prediction/proto/lane_graph.proto";
 import "modules/prediction/proto/prediction_point.proto";
 
 message Lane {
-  // Features of all possible current lanes
+  // Features of all possible current lane-segments.
```

Comment:
I think the naming of "lane" is okay, or you can use "lane feature", instead. "lane segment" is a little bit confusing